frontend/src/metabase/lib/resize-observer.ts

```diff
@@ -1,8 +1,23 @@
+import { ResizeObserver as JuggleResizeObserver } from "@juggle/resize-observer";
+
 type ResizeObserverCallback = (
   entry: ResizeObserverEntry,
   observer: ResizeObserver,
 ) => void;
 
+// PR: https://github.com/metabase/metabase/pull/48227
+// The SDK team needs to use the juggle resize observer right now because the SDK
+// keeps overlaying errors of 'ResizeObserver loop completed with undelivered notifications'
+// which is really messing up the user experience.
+// With the window ResizeObserver, we were hitting these errors mostly on scalars and tables.
+//
+// This comes with some tradeoffs. On the SDK, there will be issues with scalars
+// not rendering properly on the first render, as we rely on rapid resize observer
+// updates to resize the text.
+const ResizeObserver = process.env.IS_EMBEDDING_SDK
```

Comment:
Sorry I missed this before it got merged, should we change the name of the const? It's technically shadowing the one in window which is a bit confusing, we had a bug some months ago because we used a const that was shadowing the `close` function from window.
PR where we disabled `close` variable name as it shadowed a global function: https://github.com/metabase/metabase/pull/46753